docs/sites.md

```diff
@@ -0,0 +1,19 @@
+# Sites and systems where EESSI is available
+
+Below we list some sites that we are aware of that support or make available EESSI to their users:
+
+* EMBL Heidelberg
+* VUB Brussels
+* HPC-UGent (7 systems + VSC Tier-1 Hortense)
+* EuroHPC: Vega and Karolina
+* BAZIS cluster Vrije Universiteit Amsterdam
+* ICER HPC resources at Michigan State University
+* Norwegian Research Infrastructure Services - Betzy
+* University of Groningen HPC Cluster ('Hábrók') and also on the university's centrally provided/managed Linux workplaces
+* SURF Snellius and Spider cluster, SURF Research Cloud
```

Comment:
Actually, it's also available on our experimental cluster (`Liza`). So it should probably read:
`SURF Snellius, Spider and Liza clusters, SURF Research Cloud`